website/src/categories/getting-started/developers.md

```diff
@@ -20,6 +20,23 @@ eleventyNavigation:
 
 <section>
 
+## Make sure you have access
+
+Even though the SL Design System is open source, the packages are not. That is why you need to get access to the npm packages in order to install them locally. First the SL Design System team needs to add you to the list of users who can access the packages. To be added to that list you can <a href="mailto:designsystem@sanoma.com">send us an email</a> or reach out to us in [our Slack channel](https://sanoma.slack.com/archives/C03SA9HUUA3).
+
+Once you are added you need to [create up a personal access token (classic) on Github](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic). Make sure that this token has at least the `read_packages` privilege. This newly created token needs to be added to your .npmrc file, along with a reference to where the `@sl-design-system` packages can be found:
+<div class="ds-code">
+
+  ```txt
+  //npm.pkg.github.com/:_authToken=<token you just generated>
```

Comment:
Provide a small explainer with the best practice to put your own `authToken` in the `~/.npmrc` (home directory) and not in any project folder.

Reply:
Something like "Make sure you add the token to `~/.npmrc`, the file in your home directory, not in a project folder. That way you don't run the risk of accidentally committing and pushing your secret token."?